### DISCUSSION TITLE
PyMeshLab 2021.10

### DISCUSSION THREAD

**Discussion by alemuntoni** [cnr-isti-vclab/PyMeshLab#160] [General]
There is a new version of PyMeshLab ready to be downloaded from PyPi! :)
You can check the changelog describing what's new w.r.t. the old version [here](https://pymeshlab.readthedocs.io/en/latest/changelog.html).

Starting from the next PyMeshLab version, almost all the filters will be renamed according to what is the main goal of each filter, making more clear what a filter does. We will provide scripts that will replace old names with new names automatically, in order to make this change less painful as possible.

**Comment by jmespadero** [comment]
Congratulations and thanks for all the work done in this new version.

grazie mille.
